tests/test_models/test_detectors/test_deformable_detr.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from unittest import TestCase
+
+import torch
+from mmengine.config import ConfigDict
+from mmengine.structures import InstanceData
+
+from mmdet.models.detectors.deformable_detr import DeformableDETR
+from mmdet.structures import DetDataSample
+from mmdet.utils import register_all_modules
+
+
+class TestDeformableDETR(TestCase):
+
+    def setUp(self):
```

Comment:
always use low case  letters for the function name

Reply:
We use function name “setUp” the same as  “https://github.com/Li-Qingyun/mmdetection/blob/3c016c823a5ba4eebb9a0a318461104b0f669732/tests/test_models/test_detectors/test_single_stage.py#L17”。
class TestSingleStageDetector(TestCase):

    def setUp(self):
        register_all_modules()

Reply:
just fix them togather

Reply:
![9575e6b1e129b791b0963b49b5e30c5](https://user-images.githubusercontent.com/70597027/197403246-f801dfb3-ea66-4867-9aa4-9a28249c24f6.jpg)
It is the style of unittest lib,as the function setUp here just overwrote the function setUp in ubittest lib,I think maybe we should not change it.